From: Daniel Drake <dsd@laptop.org>

Fix build against metacity-2.34.2. The port to GSettings
removed these constants from the codebase.

Index: gnome-python-desktop-2.32.0/metacity/metacity.defs
===================================================================
--- gnome-python-desktop-2.32.0.orig/metacity/metacity.defs
+++ gnome-python-desktop-2.32.0/metacity/metacity.defs
@@ -120,32 +120,6 @@
   )
 )
 
-(define-enum FocusMode
-  (in-module "Meta")
-  (c-name "MetaFocusMode")
-;   (gtype-id "META_TYPE_FOCUS_MODE")
-  (values
-    '("click" "META_FOCUS_MODE_CLICK")
-    '("sloppy" "META_FOCUS_MODE_SLOPPY")
-    '("mouse" "META_FOCUS_MODE_MOUSE")
-  )
-)
-
-(define-enum ActionTitlebar
-  (in-module "Meta")
-  (c-name "MetaActionTitlebar")
-;   (gtype-id "META_TYPE_ACTION_TITLEBAR")
-  (values
-    '("toggle-shade" "META_ACTION_TITLEBAR_TOGGLE_SHADE")
-    '("toggle-maximize" "META_ACTION_TITLEBAR_TOGGLE_MAXIMIZE")
-    '("minimize" "META_ACTION_TITLEBAR_MINIMIZE")
-    '("none" "META_ACTION_TITLEBAR_NONE")
-    '("lower" "META_ACTION_TITLEBAR_LOWER")
-    '("menu" "META_ACTION_TITLEBAR_MENU")
-    '("last" "META_ACTION_TITLEBAR_LAST")
-  )
-)
-
 (define-enum FrameType
   (in-module "Meta")
   (c-name "MetaFrameType")
